cms/djangoapps/contentstore/views/preview.py

```diff
@@ -124,6 +125,28 @@ def user_id(self):
         return self._request.user.id
 
 
+class StudioPermissionsService(object):
```

Comment:
It's only used in Studio, so it doesn't really need to be shared... It's also only an interim service intended to be replaced by whatever future standard service there is to make XBlocks aware of users & permissions. That said, I can move the code wherever you want, as long as everything will connect properly. I had tried a different approach before this one, but it didn't work because `student.auth` is a Django app, and this block lives in `common/lib` which can't have Django dependencies.